src/jit/compiler.h

```diff
@@ -487,6 +487,45 @@ class LclVarDsc
         assert(_lvRegNum == reg);
     }
 
+    const size_t lvArgStackSize() const
+    {
+        // Make sure this will have a stack size
+        assert(!this->lvIsRegArg);
+
+        size_t stackSize = 0;
+        if (varTypeIsStruct(this))
+        {
+#if defined(WINDOWS_AMD64_ABI)
+            // StackSize is always the pointer size.
```

Comment:
StackSize is always the pointer size. => Structs are either passed by reference or can be passed by value using one pointer,